app/controllers/api/v1/users/devices_controller.rb

```diff
@@ -25,4 +25,19 @@ def show
     respond_with list
   end
 
-end
\ No newline at end of file
+  def update
```

Comment:
I think we should check that the user is signed in and that the signed in user owns the device being updated. I have implemented this in some methods on my branch so might be easier to merge the pr, then for me to merge dev into my branch and make changes?